tests/test_mdspan_swap.cpp

```diff
@@ -0,0 +1,123 @@
+
+#include <experimental/mdspan>
+#include <vector>
+
+#include <gtest/gtest.h>
+#include "offload_utils.hpp"
+#include "foo_customizations.hpp"
+
+namespace stdex = std::experimental;
+_MDSPAN_INLINE_VARIABLE constexpr auto dyn = stdex::dynamic_extent;
+
+
+void test_mdspan_std_swap_static_extents() {
+  size_t* errors = allocate_array<size_t>(1);
+  errors[0] = 0;
+
+  dispatch([=] _MDSPAN_HOST_DEVICE () {
+    int data1[12] = {1,2,3,4,5,6,7,8,9,10,11,12};
```

Comment:
Would you consider making the data differ, and testing that the swap was shallow?

Reply:
sure

Reply:
actually not sure why that would be different from the pointer comparison

Reply:
done